experimental/packages/sdk-logs/src/index.ts

```diff
@@ -0,0 +1,5 @@
+export * from './Logger';
+export * from './LoggerProvider';
+export * from './LogRecordProcessor';
+export * from './export/SimpleLogProcessor';
+export * from './export/ConsoleLogExporter';
```

Comment:
Explicit named exports would make it so that we don't accidentally leak internals. We've been moving that way in other packages.